### PR TITLE
Microverse To-do list

### PR DESCRIPTION
**Pull request by ha-manel**
## In this project: 
- I created the HTML elements for the to-do list.
- I styled the to-do list using CSS3.
- I created JavaScript modules for adding, removing, editing, and completing tasks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+## Requirements
+
+Build a Todo list using:
+
+- HTML5
+- CSS3
+- JavaScript (ES6)
+- Webpack
+- Linters (Lighthouse, Webhint, Stylelint, Eslint)
+
```
Comment (RWUBAKWANAYO) [inline README.md:16]: - [ ]  Kindly add font-awesome to the stack used to create a project in the readme file

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<body>
+  <main>
+    <p id="header">Today's To Do <button id="refresh-list"><i class="fa-solid fa-arrows-rotate"></i></button></p>
+
+    <input type="text" id="new-task" placeholder="Add to your list...">
+
+    <ul id="todo-list">
+    </ul>
+
+    <p id="clear-completed">Clear all completed</p>
```
Comment (RWUBAKWANAYO) [inline dist/index.html]: - [ ] Please replace `<p>` tags used used at line 14 and line 21, with collect HTML tag

**Review by RWUBAKWANAYO — CHANGES_REQUESTED** [review]
Hi @ha-manel,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!


68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966

### To Highlight💪🏻
✔️ Nice file structure.
✔️ Proper Gitflow
✔️ your source code is clean 
👍 Good job!

## STATUS: CHANGES REQUESTED ♻️
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rwubakwanayo in your question so I can receive the notification.

File: dist/index.html (hunk 2 above)
Comment (ha-manel) [reply dist/index.html]: thank you, I will change it 👍🏻

File: README.md (hunk 1 above)
Comment (ha-manel) [reply README.md:16]: Thank you, I forgot about it ! I will add it 😊

**Review by RWUBAKWANAYO — APPROVED** [review]

**Pull request merged**